Documentation/SwiftlyDocs.docc/automated-install.md

```diff
@@ -0,0 +1,66 @@
+# Install Swiftly Automatically
+
+Swiftly can be installed automatically in places like build/CI systems.
+
+This guide will help you to script to the installation of swiftly and toolchains so that it can be unattended. We assume that you have working understanding of your build system. The examples are based on a typical Unix environment.
+
+First, download a swiftly binary from a trusted source, such as your artifact repository, or a well-known website for the operating system (e.g. Linux) and processor architecture (e.g. arm64, or x86_64). Here's an example using the popular curl command.
+
+```
+curl -L <trusted_location_of_swiftly> > swiftly
+```
+
+> Tip: If you are using Linux you will need the "ca-certificates" package for the root certificate authorities that will establish the trust that swiftly needs to make API requests that it needs. This package is frequently pre-installed on end-user environments, but may not be present in more minimal installations.
+
+Once swiftly is downloaded you can run the init subcommand to finish the installation. This command will use the default initialization options and proceed without prompting.
+
+```
+./swiftly init --assume-yes
+```
+
+Swiftly is installed, but the current shell may not yet be updated with the new environment variables, such as the PATH. The init command prints instructions on how to update the current shell environment without opening a new shell.
+
+```
+To begin using installed swiftly from your current shell, first run the following command:
+
+. "/root/.local/share/swiftly/env.sh"
```

Comment:
Need to make it clear this is for Linux, maybe provide macOS version as well

Reply:
`env.sh` is not accurate, for `fish`, it'll be `env.fish`.

Reply:
> `env.sh` is not accurate, for `fish`, it'll be `env.fish`.

Right, I'm hoping to make this an illustrative example of something that an integrator would look for from the swiftly output so that they can automate it. I'll see if I can make the description above highlight that this is just an example and can be different depending on the environment (e.g. Windows or Fish)